Hello/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>typer</title>
+	<meta charset="utf-8" />
+	<link rel="stylesheet" type="text/css" href="app.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
+	<script src="app.ts"></script>
```

Comment:
I tried it in chrome and it worked. Im not very familiar with ts, tell me more about it.